perf/specjbb/build/getSpecjbbSuite.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+
+SUCCESS=0
+ERROR=1
+
+exists() {
+    local specjbb_dir=$1
+    return $(test -f "${specjbb_dir}/${SPECJBB_JAR}")
+}
+
+isRunnable() {
+    local specjbb_dir=$1
+    #result=$(${JAVA_BIN} -jar ${specjbb_dir}/${SPECJBB_JAR} -v > /dev/null 2>&1 | grep -q "SPECjbb2015") > /dev/null 2>&1
+    result=$(${JAVA_BIN} -jar ${specjbb_dir}/${SPECJBB_JAR} -v | grep -q "SPECjbb2015") > /dev/null 2>&1
+    return $result
+}
+
+dirIsValid(){
+    local specjbb_dir=$1
+
+    if exists $specjbb_dir; then
+        if isRunnable $specjbb_dir; then
+            return $SUCCESS
+        fi
+    fi
+
+    return $ERROR
+}
+
+dstIsValid(){
+    echo "##### Validating destination suite"
+    return $(dirIsValid $SPECJBB_DEST)
+}
+
+srcIsValid(){
+    echo "##### Validating source suite"
+    return $(dirIsValid $SPECJBB_SRC)
+}
+
+syncSrcAndDst(){
+    echo "##### Syncing destination with source"
+    rsync -a $SPECJBB_SRC/ $SPECJBB_DEST
```

Comment:
The trailing slash is not required for the dest path; the slash was included for the src path because it modified the behavior of rsync. If the slash was excluded from the src path then the command would copy the directory instead of the contents of the directory. I'll add that info as a comment in the code.